go-controller/pkg/config/config.go

```diff
@@ -35,6 +35,8 @@ const DefaultAPIServer = "http://localhost:8443"
 // Default IANA-assigned UDP port number for VXLAN
 const DefaultVXLANPort = 4789
 
+const DefaultDBTxnTimeout = 100
```

Comment:
Any reason for not doing this skipping the GetDbTxnTimeout func?
```suggestion
var DefaultDBTxnTimeout = 100 * time.Second
```

Reply:
Mostly because it is a default value for config flag of type uint

Reply:
It should be straightforward to use time.Duration instead, we are doing it here:
https://github.com/ovn-org/ovn-kubernetes/blob/c04b0861edbd7e8154296a271b3af5da47c4e646/go-controller/pkg/config/config.go#L345